web/forms.py

```
@@ -1155,6 +1156,36 @@ class ForumTopicForm(forms.Form):
             }
         ),
     )
+    github_issue_url = forms.URLField(
+        required=False,
+        help_text="Link to a related GitHub issue (optional)",
+        widget=forms.URLInput(
+            attrs={"placeholder": "https://github.com/alphaonelabs/alphaonelabs-education-website/issues/123"}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)